### PR TITLE
Add config to override index page

### PR DESCRIPTION
**Pull request by andrewm89**

### CODE REVIEW

File: invenio_communities/views/ui.py [hunk 1]
```diff
@@ -189,6 +191,12 @@ def index():
     page = max(page, 1)
     p = Pagination(page, per_page, len(communities))
 
+    try:
+        facets = CommunitiesFacets().get()['communities']['buckets']
+    except:
```
Comment (audub) [inline invenio_communities/views/ui.py]: Can we do a more specific exception?

File: invenio_communities/views/ui.py [hunk 2]
```diff
@@ -189,6 +191,12 @@ def index():
     page = max(page, 1)
     p = Pagination(page, per_page, len(communities))
 
+    try:
+        facets = CommunitiesFacets().get()['communities']['buckets']
+    except KeyError:
+        facets = None
+        pass
```
Comment (jlebonzec) [inline invenio_communities/views/ui.py]: You don't need `pass` anymore, since you do something. :)

**Comment by jlebonzec** [comment]
Minor comment, then green light! :)

**Comment by blixhavn** [comment]
:shipit:

**Pull request merged**